identity_data/src/commonMain/kotlin/net/thechance/mena/identity/data/mapper/AddressMapper.kt

```diff
@@ -0,0 +1,12 @@
+package net.thechance.mena.identity.data.mapper
+
+import net.thechance.mena.identity.data.dto.addresses.AddressRequestDto
+import net.thechance.mena.identity.domain.entity.Address
+
+fun Address.toAddressRequestDto(): AddressRequestDto = AddressRequestDto(
```

Comment:
I don't think it's a compact function, and suggest it to be like `toDto()` instead
```suggestion
fun Address.toDto(): AddressRequestDto {
    return AddressRequestDto(
        latitude = latitude,
        longitude = longitude,
        addressType = addressType,
        addressLine = addressLine,
        otherAddressType = otherAddressType
    )
}
```